Formula/percona-toolkit.rb

```diff
@@ -53,10 +53,12 @@ def install
     # Disable dynamic selection of perl which may cause segfault when an
     # incompatible perl is picked up.
     # https://github.com/Homebrew/homebrew-core/issues/4936
-    non_perl_files = %w[bin/pt-ioprofile bin/pt-mext bin/pt-mysql-summary
-                        bin/pt-pmp bin/pt-sift bin/pt-stalk bin/pt-summary]
-    perl_files = Dir["bin/*"] - non_perl_files
-    inreplace perl_files, "#!/usr/bin/env perl", "#!/usr/bin/perl"
+    Pathname(bin).find do |f|
```

Comment:
Isn't `bin` already a `Pathname`? Does `bin.find` not work?

Reply:
You're right, thanks!